app/javascript/lib/validation/expression-matrices-validation.js

```diff
@@ -354,7 +358,7 @@ function validateValuesAreNumeric(line, isLastLine, lineNum, dataObj) {
   // skip first column
   const lineWithoutFirstColumn = line.slice(1)
 
-  if (lineWithoutFirstColumn.some(isNaN)) {
+  if (lineWithoutFirstColumn.some(isEmptyOrNonNumeric)) {
```

Comment:
since this is going to run on every value in the file, I suspect we'll want to inline it as a for loop, rather than using 'some' and a function.  But I don't know how much performance gains that gives us